sig-scalability/README.md

```diff
@@ -26,20 +26,21 @@ For more details about our objectives please review [Scaling And Performance Goa
 - [kubernetes-sig-scale](https://groups.google.com/forum/#!forum/kubernetes-sig-scale)
 
 ## Docs
+- [Extending SLO definitions](extending_slo.md)
 - [Scaling And Performance Goals](goals.md)
 
-## Scalability SLAs
+## Scalability SLOs
 
-We officially support two different SLAs:
+We officially support two different SLOs:
 
 1. "API-responsiveness":
    99% of all API calls return in less than 1s
 
 1. "Pod startup time:
    99% of pods (with pre-pulled images) start within 5s
 
-This should be valid on appropriate hardware up to a 1000 node cluster with 30 pods/node.  We eventually want to expand that to 100 pods/node.
+This should be valid on appropriate hardware up to a 5000 node cluster with 30 pods/node.  We eventually want to expand that to 100 pods/node.
 
 For more details how do we measure those, you can look at: http://blog.kubernetes.io/2015_09_01_archive.html
 
-In the future we may want to add more SLAs (e.g. scheduler throughput), but we are not there yet.
+In the future we may want to add more SLOs (e.g. scheduler throughput). 
```

Comment:
We are working on refining existing SLOs and defining more for other areas of the system.